prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -244,6 +253,32 @@ class FHIRRouter(
         }
     }
 
+    private fun findReceiversForBundle(
```

Comment:
I wasn't picturing we would need to do anything with the route step. It seems like this check to make sure the original format matches the receiver's accepted format could be done more cleanly in the translate step?

I think its cleaner if route step just check route filters. If original passthrough is enabled and it doesn't match receiver's specifications then that sounds like a translate step problem since when we are in the route step we don't care about format, just the data.

Reply:
Here is where that decision is being made as to who to send it to, so I think it makes the most sense to do it here. I am not sure how I would do this in the translate step.

Reply:
Yes the decision who to send it to is done here, and format is not a consideration that the routing step takes into account. It only worries about the configuration of the routing filters.

That said, can't you basically delete all changes in this function and in translate just do something like:

```
if (receiver.format == message.originalReportFormat) {
    getOriginalMessage(message.originalReportId as ReportId, WorkflowEngine())
} else {
    // log err via action log
    // throw exception
}
```

That seems a lot cleaner to me and now the translate function is doing what its supposed to do: make sure the format of the message is in a format the receiver can accept, otherwise throw an error.